### PR TITLE
Provide sequence diagrams for Device Registration API description.

### PR DESCRIPTION
**Pull request by sysexcontrol**
Signed-off-by: Karsten Frank <Karsten.Frank@bosch-si.com>

### CODE REVIEW

File: site/diagrams/sequenceDiagrams/assertDeviceRegistration_Success.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+title Assert Device Registration (success)
+
+Client->+Device Registration Service: transfer("registration/tenant", "assert", device-id)
+note over Device Registration Service:
+lookup device and check enabled flag
+generate assertion
+end note
+Device Registration Service->Client: transfer("registration/tenant/${reply-to}", device-id, assertion)
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/assertDeviceRegistration_Success.txt]: maybe we should also include the status code (200) in this case? we do include it in all other interactions.

File: site/diagrams/sequenceDiagrams/getRegistrationInformation_Success.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+title Get Registration Information (success)
+
+Client->+Device Registration Service: transfer("registration/TENANT", "get", device-id)
+
+note over Device Registration Service:
+lookup device
+end note
+
+Device Registration Service->-Client: transfer("registration/TENANT/${reply-to}", device-id, enabled, metadata)
```
Comment (sophokles73) [inline site/diagrams/sequenceDiagrams/getRegistrationInformation_Success.txt]: status code?

File: site/diagrams/sequenceDiagrams/assertDeviceRegistration_Success.txt (hunk 1 above)
Comment (sysexcontrol) [reply site/diagrams/sequenceDiagrams/assertDeviceRegistration_Success.txt]: Thought about this already myself, so this sounds like a good point!
Will include it here and the Credentials API diagrams, too.

**Comment by sysexcontrol** [comment]
Will do the update for Credentials API in separate commit after this PR is done (to keep commits separated).
@sophokles73 : can you have a look again, please?

**Pull request merged**